src/atom_finder/classifier/curly-braces.clj

```diff
@@ -0,0 +1,30 @@
+(in-ns 'atom-finder.classifier)
+(import '(org.eclipse.cdt.core.dom.ast IASTCompoundStatement IASTIfStatement IASTForStatement IASTWhileStatement))
+
+
+(defn curly-braces?
+  "Does this note represent curly brace?"
+  [node]
+  (instance? IASTCompoundStatement node))
+  
+
+(defn curly-braciable-statement?
+  "Does this AST node an statement that should be followed by curly braces?"
+  [node]
+  (let [statement #{;statement that should be followed by compound statements (curly braces)
+                    IASTIfStatement IASTForStatement IASTWhileStatement}]
+      (cond
+        (leaf? node) false
+        (some #(instance? % node) statement) (not-any? true? (map curly-braces? (children node)))
+        :else false)))
+          
+(def curly-braces-atom? curly-braciable-statement?)
+                                               
+
+(defn curly-braces-atoms
+  "Return all AST nodes that are missing curly braces"
+  [node]
+  (cond
+    (leaf? node) nil
+    (curly-braciable-statement? node) [node]
+    :else (mapcat curly-braces-atoms (children node))))
```

Comment:
For these lines I mostly copied from logic-as-control-flow.clj. Can you explain how default-finder works and the reason you used these lines for logic-as-control-flow?